### PR TITLE
W5/latest

### PR DESCRIPTION
**Pull request by TomasBabkine-DiCaprio**

### CODE REVIEW

File: go-minitwit/db_methods.go [hunk 1]
```diff
@@ -60,11 +63,24 @@ type Follower struct {
...
+}
+
+func (Message) TableName() string {
+	return "message"
+}
+
+func (Follower) TableName() string {
+	return "follower"
+}
+
```
Comment (Ghmeding) [inline go-minitwit/db_methods.go]: We do not need these if we just define a naming strategy as in the connect_dev_DB.

File: go-minitwit/db_methods.go [hunk 2]
```diff
@@ -75,6 +91,18 @@ func connect_DB(dsn string) (*gorm.DB, error) {
 	return db, nil
 }
 
+func connect_prod_DB() (*gorm.DB, error) {
```
Comment (Ghmeding) [inline go-minitwit/db_methods.go:82]: then we can add the auto-migration here

File: go-minitwit/main.go [hunk 3]
```diff
@@ -26,7 +26,7 @@ func main() {
```
Comment (Ghmeding) [inline go-minitwit/main.go:29]: i had to add godotenv.Load() to load the env variables

**Review by Ghmeding — CHANGES_REQUESTED** [review]

File: go-minitwit/db_methods.go (hunk 2 above)
Comment (TomasBabkine-DiCaprio) [reply go-minitwit/db_methods.go:82]: Did you test this locally? I tried it yesterday and the app wouldn't build for me, but maybe I did it wrong

Comment (Ghmeding) [reply go-minitwit/db_methods.go:82]: Yep, tested and it works :))

Comment (TomasBabkine-DiCaprio) [reply go-minitwit/db_methods.go:82]: Perfect, then thanks! Maybe you can explain to us how this gorm stuff works later on, I'm still confused about it :)

Comment (Ghmeding) [reply go-minitwit/db_methods.go:82]: me too :')

**Review by Ghmeding — APPROVED** [review]

**Pull request merged**